KomaMRIIO/src/KomaMRIIO.jl

```diff
@@ -2,8 +2,6 @@ module KomaMRIIO
 
 using KomaMRICore
 using Scanf, FileIO, HDF5, MAT, JLD2    # IO related
-using MRIBase                           # At least for EncodingCounters()
-using MRIFiles                          # At least for Limits()
```

Comment:
MRIFiles must be in IO

Reply:
Addressed [here](https://github.com/cncastillo/KomaMRI.jl/commit/93dc116159040816425b02b051360e7843954068).